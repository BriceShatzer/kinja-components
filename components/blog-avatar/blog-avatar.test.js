import * as React from 'react';
import { render } from 'enzyme';

import BlogAvatar from './blog-avatar';

describe('<BlogAvatar />', () => {
	it('should render avclub avatar', () => {
		expect(render(<BlogAvatar name="avclub" />)).toMatchSnapshot();
	});

	it('should render avclub avatar with the is-anchored class', () => {
		expect(<BlogAvatar name="avclub" anchor/>).toMatchSnapshot();
	});

	it('should render avclub avatar with forced 100px square dimensions', () => {
		expect(<BlogAvatar name="avclub" size={100}/>).toMatchSnapshot();
	});
});
