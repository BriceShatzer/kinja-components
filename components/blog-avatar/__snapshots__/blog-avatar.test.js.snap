// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlogAvatar /> should render avclub avatar 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: inherit;
  height: inherit;
}

.c0 .SVGInline-svg {
  display: none;
}

.c0 svg {
  max-width: 100%;
  width: inherit;
  height: inherit;
}

<span
  class="c0"
>
  <svg
    aria-label="TheAVClub avatar"
    height="64"
    viewBox="0 0 64 64"
    width="64"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      fill="none"
      fill-rule="evenodd"
    >
      <path
        d="M0 64h64V0H0"
        fill="#1C263C"
      />
      <path
        d="M42.66 49.29l-14.8-32.24 7.05.05 7.6 17.92 7.73-17.78 6.48.05H57l-14.34 32zM22.38 32.24l3.38 7.62-7.04-.14 3.66-7.48zM7 48.91l6.82.14 1.88-4.07 12.87.26 1.6 3.94 6.78.06L22.68 17 7 48.9z"
        fill="#FFF"
      />
    </g>
  </svg>
</span>
`;

exports[`<BlogAvatar /> should render avclub avatar with forced 100px square dimensions 1`] = `
<BlogAvatar
  name="avclub"
  size={100}
/>
`;

exports[`<BlogAvatar /> should render avclub avatar with the is-anchored class 1`] = `
<BlogAvatar
  anchor={true}
  name="avclub"
/>
`;
